Source/Bogus/DataSets/Date.cs

```diff
@@ -20,8 +20,18 @@ public partial class Date : DataSet
       /// Setting this static value will effect and apply to all Faker[T], Faker,
       /// and new Date() datasets instances.
       /// </summary>
+      //[Obsolete("Please use .LocalSystemClock instead. Date.LocalSystemClock property per Date dataset instance. Setting a global static system clock is deprecated.")]
```

Comment:
TODO: Torn between obsoleting the global static `Date.SystemClock` and not.

Reply:
I think obsolete is the best choice. First it works with the static and if the code evolves, this would be an issue.

Note: if we obsolete this, then it's nice to bump at least the minor version (e.g. 34.1 or 35.0) - but maybe that was already the plan

Reply:
Would it be replaced then simply by a default function returning DateTime.Now?